.docker/scripts/serve.sh

```diff
@@ -5,11 +5,15 @@ dockerize -wait tcp://postgres:5432 -timeout 1m
 dockerize -wait tcp://solr:8983 -timeout 1m
 dockerize -wait tcp://redis:6379 -timeout 1m
 
-sed -i "s@SITE_URL@${SITE_URL}@g" $CKAN_INI
+for i in {1..60}; do
```

Comment:
This syntax is actually not valid (my bad), see ckanext-qgov for a correct way of doing it.